vl3-release-note.md

```diff
@@ -0,0 +1,89 @@
+## New Features & Enhancements 
+
+- Migrated to Vega 5 
+
+- Composite Marks
+  - New `boxplot` and `errorbar` marks
+
+- Multi-View Composition 
+  - Support for wrapped `facet`, `concat`, and `repeat`
+  - Add `config` for `facet`, `concat`, and `repeat`
+  - Single-View specifications with `row`/`column` channels (for faceting) now support `resolve`
+  - Improve default layouts
+  - Support title anchor for multi-view composition
+
+- Sorting
+  - Make sort `"op"` optional and make `"mean"` the default op (#4475)
+  - Support sorting by another encoding (#4478)
+
+- Interactions
+  - Support `init`ializing selection (#4139)
```

Comment:
We can also read selections with 3.0 via `view.signal("selectionName")`.

Did we want to add a `vl.selection(view, "name")` (and `vl.addSelectionListener` and `vl.removeSelectionListener`) functions as discussed on our core call today, @kanitw? Or push that out?

Reply:
I don't mind adding this tomorrow since it's probably just ~10-20 LOCs.

cc: @domoritz 

I guess you prefer vl.add/removeSelectionListener to vl.selection.add/removeListener?  (I agree that flattened is better, but wanna bring 
it up just it case.)

https://github.com/vega/vega-lite/issues/4751